docs/api.md

```diff
@@ -2065,6 +2067,16 @@ Contains the URL of the request.
 #### response.buffer()
 - returns: <Promise<[Buffer]>> Promise which resolves to a buffer with response body.
 
+#### response.fromDiskCache
```

Comment:
let's make them functions to unify our API

Reply:
done